articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+
+    ![Azure replication target](images/azs-browser-replication-target.png)
+
+11. In Physical Machines, select **Physical machine**. Specify the name, IP address and the operating system of each machine you want to replicate, you should have noted these down from when you set up the machine prerequisites.
+
+    * Use the internal IP address of the machine.
+
+    **If you specify the public IP address replication might not work.**
+
+    ![Azure replication machine](images/azs-browser-replication-machine.png)
+
+12. In Properties > Configure properties, select the account that the process server will use to automatically install Mobility Service on the machine. If the configuration server has yet to sync with Public Azure, no account will appear here.
+
+    ![Azure replication properties](images/azs-browser-replication-properties.png)
+
+13. In Replication settings > Configure replication settings, check that the correct replication policy is selected.
+
+    ![Azure replication settings](images/azs-browser-replication-settings.png)
+
+14. Select **Enable Replication**.
+
+15. Track progress of the Enable Protection job in Settings > Jobs > Site Recovery Jobs. After the Finalize Protection job runs, the machine is ready for failover.
+
+    * The VM will need to synchronize before a failover can take place. You can see synchronization progress of the VM in the vault.
+
+## Step 6: Run a disaster recovery drill
+
+You can run a test failover to Azure to make sure that everything's working as expected. This failover won't affect your production environment.
+
+### Verify machine properties
+
+Before you run a test failover, verify the machine properties, and make sure that they comply with Azure requirements. You can view and modify properties as follows:
+
+1. In Protected Items, select Replicated Items > VM.
+
+2. In the Replicated item pane, there's a summary of VM information, health status, and the latest available recovery points. Select Properties to view more details.
+
+3. In Compute and Network, modify settings as needed.
+
+    * You can modify the Azure VM name, resource group, target size, availability set, and managed disk settings.
+
+    * You can also view and modify network settings. These include the network/subnet to which the Azure VM is joined after failover, and the IP address that will be assigned to the VM.
+
+4. In Disks, view information about the operating system and data disks on the VM.
+
+### Run a test failover
+
+When you run a test failover, the following happens:
+
+1. A prerequisites check runs to make sure all of the conditions required for failover are in place.
+
+2. Failover processes the data using the specified recovery point:
+
+    * Latest processed: The machine fails over to the latest recovery point processed by Site Recovery. The time stamp is shown. With this option, no time is spent processing data, so it provides a low RTO (recovery time objective).
```

Comment:
Be consistent with either a colon or full stop after the name of the option

Reply:
done